### DISCUSSION TITLE
Alcune ambiguita'

### DISCUSSION THREAD

**Discussion by OrazioCostanzo** [italia/ansc#995] [Supporto servizi cooperativi]
Buongiorno, ho alcuni dubbi da togliere:

Vorrei sapere cosa contengono questi dati dal momento che devono essere forniti in qualche modo o dall'operatore oppure devono essere ricavati dal backend:
- 11214000  Atto notarile di consenso | Estremi documento | estremiDocumento
- 12214300  Dati generali | Presentazione documenti | datiProdotti
- 12214300  Autorizzazione al riconoscimento | Estremi documento | estremiDocumento

In questo caso, qual è la differenza tra "nazioneEnte" e "nomeNazioneEnte"? di solito il primo si riferisce all'id e il secondo alla descrizione dello stato.
- 1312  Autorità mittente | Stato ente  | nazioneEnte
- 1312 Autorità mittente | Stato ente - Descrizione | nomeNazioneEnte

nel caso 1399 non capisco a cosa si riferiscono i flag flagDichiarante e flagComparente in questo contesto.

Grazie.

**Comment by acolosimosogei** [comment]
Salve, per rispondere alla prima domanda I dati per i campi di cui si parla devono essere forniti dall'operatore.

Di seguito riporto i dubbi riportati nella discussion seguiti dalla risposta.

	1) 11214000 Atto notarile di consenso | Estremi documento | estremiDocumento 

Nel campo estremi documento va compilato con l'identificativo dell'atto notarile.

	2) 12214300 Dati generali | Presentazione documenti | datiProdotti

Il campo Presentazione Documenti si presenta come un menù a tendina nel quale possono essere scelti tre valori: prodotti, acquisiti, visionati. In base al valore scelto cambierà di conseguenza la composizione della minuta dell'atto in base alla FORM. 60, viene riportata di seguito la frazione della minuta impattata dalla scelta del campo discusso; "**A dimostrazione che nulla osta al presente riconoscimento i dichiaranti mi hanno prodotto (ovvero: e/o acquisito; ovvero: e/o ho visionato) i seguenti documenti (elencarli; fra essi deve essere  compresa la copia integrale dell'atto di nascita del figlio che si riconosce quando questi sia nato in  altro Comune). Detti documenti prodotti (e/o acquisiti), muniti del mio visto, inserisco nel volume  degli allegati a questo registro**".

	3) 12214300 Autorizzazione al riconoscimento | Estremi documento | estremiDocumento

Il campo "Autorizzazione al riconoscimento" diventa obbligatorio solo per il caso di figlio incestuoso, il caso di figlio incestuoso si caratterizza da un flag che compare nella scelta del caso d'uso.

	4) In questo caso, qual è la differenza tra "nazioneEnte" e "nomeNazioneEnte"? di solito il primo si riferisce all'id e il secondo alla descrizione dello stato.

"nazioneEnte" corrisponde all'id dello stato e "nomeNazioneEnte" corrisponde allo stato.


	5) nel caso 1399 non capisco a cosa si riferiscono i flag flagDichiarante e flagComparente in questo contesto.

Il caso 1399 è un caso d'uso di servizio che lascia maggiore libertà di scelta e che serve a racchiudere un maggior numero di casistiche in modo da gestire le sezioni come meglio ritengono gli usc. Inoltre i flagDichiarante e flagComparente sono utilizzabili nel caso in cui fanno il padre, la madre e l'intestatario sono Dichiaranti o Comparenti, inoltre tutto il campo "Madre" e il campo "Padre" sono interamente opzionali.

Chiudiamo pertanto la discussion: in caso di ulteriori richieste si prega di aprirne un'altra dedicata.
Grazie
